starter/spring-boot-starter-rabbitmq/build.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2012-present the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+	id "org.springframework.boot.starter"
+}
+
+description = "Starter for using Spring AMQP and Rabbit MQ"
```

Comment:
Can we be more specific, e.g. : `Starter for using Spring AMQP with Rabbit MQ over AMQP 0.9.1 protocol`?

And probably something similar in existing started to emphasize that it is about AMQP 1.0 protocol.